### ISSUE TITLE
[EN][Achievement]Moon Guard

### ISSUE THREAD

**Issue by Samineh** [Atlantiss/BugTracker#9605]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: The [achievement](http://www.wowhead.com/achievement=5995/moon-guard) is broken.

**Current behaviour**: If your healer does not take damage at all and everyone survives the adds before [tyrande](http://www.wowhead.com/npc=54544/echo-of-tyrande), you won't get credited for it. I've done it in two ways: 1) Stayed healer didn't heal at all; didn't take even one hit; didn't get the achievement.
2) Used a shadow priest as our healer he took damage in his shadow spec, didn't get the achievement.

**Expected behaviour**: The first way to do this achievement will be having a blood dk or a strong tank to take all the adds and your ranged dps start attacking them after he takes them to the moonlight and takes all the aggro. This way only the tank will take damage and he can heal himself or use some cooldowns to survive. The other way ,as written in the [first comment](http://www.wowhead.com/achievement=5995/moon-guard#comments) in wowhead, will be done even without someone using a healing spec and if all the players in the group take damage then you must get the achievement.
**Steps to reproduce the problem**:

1. Have a strong group or have not healer.
2. Engage the adds before Echo of Tyrande.
3. Survive the adds and make sure your heal won't take damage at all (or simply don't have a healer). You will see the achievement will be bugged.

**Revision**: 5534

**Comment by Psychobird7** [comment]
Can confirm "1)"; he did not heal a single time during the entirety of the "gauntlet".  

Consequently he did not get hit at all, yet we did not get an achievement.

**Comment by Llagas** [comment]
Same here 😭 
Please fix this achievement 👼

**Comment by 1Andas** [comment]
Yeah. I am thinking it's bugged as well. I tried several methods with my friends. 
1. We tried with druid healer staying in stealth - didn't work.
2. We tried with our healer changing spec to non-healer - didn't work.
3. We tried with healer barely healing and not taking part in battles basically, keeping a close eye on him, still didn't work.

**Comment by poleloa** [comment]
wow i too can't do it.
And i have rarely good healer. Nevermind,if i'm in tank setup or dps setup.
Maybe if u,samineh in game write ticket ,then that achi will fix.

**Comment by Aviannaqt** [comment]
hi
